### ISSUE TITLE
Levels 16 and 17: More specific scenarios

### ISSUE THREAD

**Issue by charlesdevops** [flukeout/restaurant#121]
Level 16 can be solved just by writing "plate apple, plate pickle", while Level 17 with "apple, pickle". 

So it's not clear **why** should we be using pseudo-selectors.

**Comment by flukeout** [comment]
@CarloMartini It's hard to make it so that just ONE pseudo-selector (for example) can solve a level, there are many different selectors that can solve individual levels. What I'm thinking about doing is..
- Forcing the player to use the current level's selector to win
- Preventing using multiple selectors in one attempt

However, @manscisce suggested that that would limit people from testing different things out and having a kind of sandbox to try selectors in. I'm not sure how I'll solve his yet. Maybe with a 'strict' mode and a 'sandbox' mode?

What are your thoughts?

**Comment by charlesdevops** [comment]
Thanks for the answer, @flukeout !

My issue is more "didactic" than technical. If CSS Diner is used to learn selectors, I believe scenarios should be laid out so that choosing a certain selector seems a quite logical and simple step, and the user gets to actually understand why those selectors were invented in the first place... Even if, of course, there will always be multiple possible choices. At least in those two levels, I couldn't grasp the utility of the pseudo-selectors.

Anyway, regarding the solution of the levels, how about a middle ground, like printing a simple message like "Your choice work, but to pass the level please use the described selector".

**Comment by flukeout** [comment]
Yeah, or having two-tier 'completed' status. You can clear the level with any selector, but you only get a 'star' if you use the one dictated by the level.

I know what you mean about the selectors seeming arbitrary sometimes, but since you can only place a few elements on the table without things getting cluttered, it's hard to mimic real-world scenarios where a fancy pseudo-selector would be the only way to go.

Do you have suggestions for those levels that would make using the pseudo-selectors more intuitive?

**Comment by go-pingu** [comment]
What is the correct solution to this level then? I think I didn't understand how this selector works for this example, as I didn't work out an answer.
